tests/Unit/Pypp/PyppFundamentals.hpp

```diff
@@ -221,7 +221,8 @@ struct FromPyObject<long, std::nullptr_t> {
 #if PY_MAJOR_VERSION == 2 && PY_MINOR_VERSION == 7
     } else if (not PyInt_Check(t) and not PyLong_Check(t)) {
 #elif PY_MAJOR_VERSION == 3
-    } else if (not PyLong_Check(t)) {
+      // clang-tidy: hicpp-signed-bitwise
```

Comment:
Not a big deal, but spacing of comment looks odd to me here and line 225

Reply:
@geoffrey4444 - I've moved these changes to #1265 . Please approve them there.